website/src/pages/guides/continuous-developer-onboarding/index.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { FunctionComponent } from 'react'
+import { PageProps } from 'gatsby'
+
+import Layout from '../../../components/Layout'
+import { useHubSpot } from '../../../hooks/hubSpot'
+import styles from './index.module.scss'
```

Comment:
Would it make sense to name this file to be something more general (`guides.module.scss` perhaps), in case styles end up getting shared between guides?

Reply:
I think naming this file `guides.module.scss` would be appropriate if the stylesheet shared styles for all guides pages, assuming there will be more. Then it would live one directory higher under `/guides`.

My naming convention thought process comes from naming the stylesheet to match the filename of the component/page name. Typically you'd see something like this with a collocated approach:

```
/Button
  index.tsx
  Button.tsx
  button.module.scss
```

or, with index, where your editor shows the directory name for convenience (see screenshot). I guess you lose search specificity in a way here.
![Screen Shot 2022-03-21 at 4 21 00 PM](https://user-images.githubusercontent.com/1733936/159357234-e5f7dbc1-3652-4461-bd0a-34e3a4abeb89.png)


```
/myPage
  index.tsx
  index.module.scss
```

or

```
/myPage
  index.tsx
  myPage.module.scss
```

Since this stylesheet is more specific to this page only, I'm happy to go with something like `onboarding.module.scss` for short, even though it doesn't match the directory name for the page URI. But just wanted to let you know why this was intentional. Let me know what you think.

Reply:
I think since this PR has created that directory under the assumption that there will be more guides it would make sense to move it up.

Reply:
I don't think that's appropriate yet since the style blocks listed in this stylesheet are only relevant to this page and `guides.module.scss` could grow as more guides pages are added. I'd prefer renaming the stylesheet to `onboarding.module.scss` as a more descriptive filename if the component/stylesheet filename matching convention isn't preferred (`index.module.scss`).

Since we don't know the designs yet for other guides pages, I think we should evolve this at that point instead of developing changes in anticipation for design specs that we don't know yet. If the styles can be shared, we could make scoped and colocated guides related components at that point instead of having a stylesheet per page for such small style blocks: (ie: GuidesHero, GuidesForm)

Reply:
Definitely let's not have a per-page guide for small blocks, totally agree. Let's leave as-is for now then.